extensions/wikia/CuratedContent/CuratedContentController.class.php

```diff
@@ -440,14 +440,24 @@ public function getData( ) {
 
 		if ( !empty( $wgWikiaCuratedContent ) && is_array( $wgWikiaCuratedContent )  ) {
 			foreach ( $wgWikiaCuratedContent as $section ) {
-				// sections
-				if ( !empty( $section['title'] ) && empty( $section['featured'] ) ) {
+				// update information about node type
+				$section['node_type'] = 'section';
```

Comment:
All the rest of properties use underscore as they come from `$wgWikiaCuratedContent` like that. Should we convert all of them? We will have to revert the process when saving.